gitlab-ci/container/Dockerfile

```diff
@@ -84,6 +84,10 @@ RUN . /opt/nvm/nvm.sh && \
     nvm install ${dependency_mgmt_nns_dapp_node_version} && \
     nvm install ${dependency_mgmt_default_node_version}
 
+# Install JRE for TLA/Apalache checks
+RUN apt -yq update && \
```

Comment:
Introducing a new layer just to add a new package is too much. I think you can add this package to `files/packages.common`.

Reply:
Thanks - moved it to packages.common now